apps/developer-hub/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@pythnetwork/developer-hub",
+  "version": "0.0.0",
+  "private": true,
+  "type": "module",
+  "engines": {
+    "node": "22"
+  },
+  "scripts": {
+    "build:vercel": "next build",
+    "fix:format": "prettier --write .",
+    "fix:lint:eslint": "eslint --fix .",
+    "fix:lint:stylelint": "stylelint --fix 'src/**/*.scss'",
+    "pull:env": "[ $CI ] || VERCEL_ORG_ID=team_BKQrg3JJFLxZyTqpuYtIY0rj VERCEL_PROJECT_ID=prj_TBkf9EyQjQF37gs4Vk0sQKJj97kE vercel env pull",
+    "start:dev": "next dev --port 3003",
+    "start:prod": "next start --port 3003",
+    "test:format": "prettier --check .",
+    "test:lint:eslint": "eslint . --max-warnings 0",
+    "test:lint:stylelint": "stylelint 'src/**/*.scss' --max-warnings 0",
+    "test:types": "tsc"
+  },
+  "dependencies": {
+    "@phosphor-icons/react": "catalog:",
+    "@pythnetwork/component-library": "workspace:*",
+    "@react-hookz/web": "catalog:",
+    "@rive-app/react-canvas": "^4.19.1",
+    "@solana/web3.js": "catalog:",
+    "bs58": "catalog:",
+    "clsx": "catalog:",
+    "cryptocurrency-icons": "catalog:",
+    "dnum": "catalog:",
+    "lightweight-charts": "catalog:",
+    "motion": "catalog:",
+    "next": "catalog:",
+    "next-themes": "catalog:",
+    "nuqs": "catalog:",
+    "react": "catalog:",
+    "react-aria": "catalog:",
+    "react-dom": "catalog:",
+    "recharts": "catalog:",
+    "shiki": "catalog:",
+    "superjson": "catalog:",
+    "swr": "catalog:",
+    "zod": "catalog:",
+    "zod-validation-error": "catalog:"
```

Comment:
You should go through these dependencies and strip out stuff you don't need; I'm pretty sure you won't need the vast majority of stuff listed here...